src/init.cpp

```diff
@@ -54,6 +54,7 @@
 #include "overlay_ordering.h"
 #include "overmap_connection.h"
 #include "overmap_location.h"
+#include "overmap_biome.h"
```

Comment:
Please use `astyle` to format all C++ files you create or modify - see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.md.

Either use `make astyle`, [VS extension](https://github.com/CleverRaven/Cataclysm-DDA/pull/24006) or run download and run `astyle` manually.

Reply:
I’ll get this done, was going to wait until a further commit but as said earlier, I may as well do sooner rather than later